src/utils/get-analytics-player-options.js

```diff
@@ -20,11 +20,11 @@ const getCloudinaryOptions = (cloudinaryOptions = {}) => ({
 });
 
 const getTranscriptOptions = (textTracks = {}) => {
-  const tracksArr = [textTracks.captions, ...textTracks.subtitles];
+  const tracksArr = [textTracks.captions, ...(textTracks.subtitles || [])];
   return {
     textTracks: hasConfig(textTracks),
     textTracksLength: tracksArr.length,
-    textTracksOptions: hasConfig(textTracks.options) || Object.keys(textTracks.options).join(','),
+    textTracksOptions: hasConfig(textTracks.options) && Object.keys(textTracks.options).join(','),
```

Comment:
how these two changes are related to exporting light version?

Reply:
They don't, just fixed a warning I got while working on this